app/Models/Configs.php

```diff
@@ -298,4 +298,12 @@ public function scopeAdmin(FixedQueryBuilder $query): FixedQueryBuilder
 	{
 		return $query->where('confidentiality', '<=', 3);
 	}
+
+	/**
+	 * Reset the cache.
+	 */
+	public static function dropCache(): void
```

Comment:
```suggestion
	public static function clearCache(): void
```

or

```suggestion
	public static function invalidateCache(): void
```

are probably the more frequently used terms.

However, I don't understand _why_ we suddenly need this method? How could the cache get out-of-sync with the DB?